src/app/garfile/responsibleperson/validations.js

```diff
@@ -31,20 +32,34 @@ module.exports.validations = (req) => {
     [
       new ValidationRule(validator.notEmpty, 'responsibleGivenName', responsibleGivenName, 'Enter a given name for the responsible person'),
       new ValidationRule(validator.isValidStringLength, 'responsibleGivenName', responsibleGivenName, `Given name must be ${MAX_STRING_LENGTH} characters or less`),
+      new ValidationRule(validator.isAlpha, 'responsibleGivenName', responsibleGivenName, `Given name must not contain special characters or numbers`),
```

Comment:
does this mean we are not accepting accents and other diacritics for names? have you discussed this with AMG?

Reply:
Yes, we are not. I discussed this with Oscar lee, here are the messages below.
In summary:
* AMG does not support non latin alphabet.
* Commerical aviation requires it to be latin alphabet.
* Other AMG downstream systems require it to be latin alphabet.


```
Hi Alistair, at the moment AMG only accepts latin alpha characters (i.e. A-Z)


Alastair Mottram-Epson
  [5:01 PM](https://homeofficedigital.slack.com/archives/D05J0FLH28G/p1698771707177339)
Cool, is there any plan to change that or not?
Naturally our service has international customers.
Also, some names maybe very hard to change into English e.g. names from different alphabet like chinese or korean names.
(edited)


Oscar Lee
  [5:05 PM](https://homeofficedigital.slack.com/archives/D05J0FLH28G/p1698771938791739)
no intention at the moment, technically we support UTF-8 so we could potentially remove the regex but for consistency currently commercial aviation use latin alphabet A-Z and we require consistency for downstream systems
[5:05](https://homeofficedigital.slack.com/archives/D05J0FLH28G/p1698771959767739)
Chinese and Korean names likely have romanised translations


Alastair Mottram-Epson
  [5:07 PM](https://homeofficedigital.slack.com/archives/D05J0FLH28G/p1698772025362019)
Ok cool.
[5:10](https://homeofficedigital.slack.com/archives/D05J0FLH28G/p1698772212809919)
I will go with the latin alphabet as the accepted version.
Out of curiousity, if someone with a chinese passport had only their chinese name down, would that prevent them from being checked for valid permission to travel by AMG or not?
If not, how does that work e.g. only the document number is relevant?
(edited)


Oscar Lee
  [5:13 PM](https://homeofficedigital.slack.com/archives/D05J0FLH28G/p1698772419739989)
I believe when they apply for an ETA they have to submit using latin alphabet characters, most travel documents I believe have romanised latin characters
[5:14](https://homeofficedigital.slack.com/archives/D05J0FLH28G/p1698772462966089)
a lot of things are used to identify a person, passport number, date of birth, document expiry date etc.


Alastair Mottram-Epson
  [5:15 PM](https://homeofficedigital.slack.com/archives/D05J0FLH28G/p1698772511731799)
Ok, cool. I did not know that chinese citizens needed an ETA to travel to the UK.
[5:15](https://homeofficedigital.slack.com/archives/D05J0FLH28G/p1698772523089999)
That makes way more sense, thanks.


Alastair Mottram-Epson
  [11:37 AM](https://homeofficedigital.slack.com/archives/D05J0FLH28G/p1698838641464269)
lot of things are used to identify a person, passport number, date of birth, document expiry date etc.
Hi Oscar, on the above, is it true that:
first names and last name don't need to be accurate in order for UPT to identify a person as valid to travel or not?
this is true for all countries?


Oscar Lee
  [11:38 AM](https://homeofficedigital.slack.com/archives/D05J0FLH28G/p1698838723866419)
not sure what is used to resolve to a person, it’s logic that is contained in another system but aware they use 8 mandatory fields
```